### ISSUE TITLE
Challenge is getting accepted without sufficient code.

### ISSUE THREAD

**Issue by akshaynaik404** [freeCodeCamp/freeCodeCamp#15726]
#### Challenge Name

https://www.freecodecamp.org/challenges/convert-json-data-to-html

#### Issue Description

Challenge is getting accepted without sufficient code.

#### Browser Information

* Browser Name, Version: Chrome Version 60.0.3112.78 (Official Build) (64-bit)
* Operating System: Ubuntu 16.04.2 LTS
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```
<script>
  $(document).ready(function() {

    $("#getMessage").on("click", function() {
      $.getJSON("/json/cats.json", function(json) {

        var html = "";
        // Only change code below this line.
        
        console.log(json)        
        json.forEach(function () {
          
        }); 
        // Only change code above this line.

        $(".message").html(html);
      });
    });
  });
</script>
```

**Comment by dhcodes** [comment]
Confirmed. We need to update the current test or add a new one.

**Comment by akshaynaik404** [comment]
I am a first time contributor, Do you think I can solve it? If yes, Any instructions?

**Comment by dhcodes** [comment]
@akshaynaik404 probably! Our tests are usually written in Regex. Set up your local instance by following this instructions: https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md

**Comment by prtk418** [comment]
@akshaynaik404 are you working on it??

**Comment by akshaynaik404** [comment]
Yes @prtk418 .

**Comment by prtk418** [comment]
@akshaynaik404 ok

**Comment by akshaynaik404** [comment]
Is this the right branch, file that I should be looking to modify @dhcodes ?

[https://github.com/freeCodeCamp/freeCodeCamp/blob/backup/master/seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json](https://github.com/freeCodeCamp/freeCodeCamp/blob/backup/master/seed/challenges/01-front-end-development-certification/json-apis-and-ajax.json)

I can see where the new test should go,
just want to confirm that its the right file.

**Comment by vivek-agrawal27** [comment]
@akshaynaik404  Yes that's the right file.

**Comment by ResamVi** [comment]
So I allowed myself to work on this as it seems this is open for Hacktoberfest.

I decided not to test whether `Object.create` was used in the code (as it's hinted in the instructions) because there are multiple ways to solve this (a for-loop as example). 

It really is difficult to test accurately but not force someone into a pttern

**Comment by raisedadead** [comment]
@ResamVi thanks for working on this and fixing the tests. 

**NOTE**
This is still open for a fix on `backup/master` branch (production site) for any one instrested.

**Comment by ahmadabdolsaheb** [comment]
@raisedadead i would like to work on this if it is still available?

**Comment by jkao1** [comment]
@ahmadabdolsaheb this challenge doesn't exist on beta; I guess it was cut out of the program.

**Comment by raisedadead** [comment]
Yup, this was removed in beta, but @ahmadabdolsaheb if interested you are welcome to open a PR against the `backup/master`.

**Comment by ahmadabdolsaheb** [comment]
@raisedadead done. 
@ResamVi thanks for the code.

**Comment by BerkeleyTrue** [comment]
production is on a feature freeze. Since this change has already been made, this issue is no longer relevant.

Happy Coding
